general/cisetup/cisetup.go

```diff
@@ -377,15 +358,24 @@ func (cc *CiSetupCommand) logCompletionInstruction(pipelineName string) error {
 	}
 
 	instructions := []string{
-		colorTitle("To complete the setup, run the following commands:"), "",
+		"", colorTitle("Completing the setup"),
+		"We configured the JFrog Platform and generated a pipeline.yml for you.",
+		"To complete the setup, add the new pipeline.yml to your git repository by running the following commands:", "",
 		"cd " + cc.data.LocalDirPath,
 		"git commit -m \"Add pipelines.yml\"",
 		"git push", "",
 		"Although your pipeline is configured, it hasn't run yet.",
 		"It will run and become visible in the following URL, after the next git commit:",
 		getPipelineUiPath(serviceDetails.Url, pipelineName), "",
-		"To create a user for your ide, that can access the builds generated by this setup, run the following command with fields of your choice:", "",
-		fmt.Sprintf(createUserTemplate, ideUserName, ideUserPassPlaceholder, ideUserEmailPlaceholder, ideGroupName), "",
+		colorTitle("Allowing developers to access this pipeline from their IDE"),
+		"You have the option of viewing the new pipeline's runs from within IntelliJ IDEA. More IDEs will be supported in the future.",
+		"To achieve this, follow these steps:",
+		" 1. Make sure the latest version of JFrog Plugin is installed on IntelliJ IDEA.",
+		" 2. Create a JFrog user for the IDE by running the following command:", "",
+		"    " + fmt.Sprintf(createUserTemplate, ideUserName, ideUserPassPlaceholder, ideUserEmailPlaceholder, ideGroupName), "",
+		" 3. In IDEA, under 'JFrog Global Configuration', set the JFrog Platform URL and the user you created.",
+		" 4. In IDEA, under 'JFrog CI Integration', set * as the 'Build name pattern'.",
+		" 5. In IDEA, open the 'JFrog' panel at the bottom of the screen, choose the 'CI' tab and then click on the 'Refresh' button.", "",
```

Comment:
Clicking on the "refresh" button is redundant since after changing the configuration, a scan is already triggered.